istio-control/istio-discovery/values.yaml

```diff
@@ -124,4 +124,21 @@ telemetry:
   v2:
     # For Null VM case now. If enabled, will set disableMixerHttpReports to true and not define mixerReportServer
     # also enable metadata exchange and stats filter.
-    enabled: false
\ No newline at end of file
+    enabled: true
```

Comment:
Can you add the master/1.5 filters? Otherwise we can't use telemetry v2 on master yet we are enabling it..

Reply:
That should go in the master branch, not 1.4

Reply:
Oh I thought this was master.

In that case this should be false

Reply:
put it back as force